modules/virt-creating-data-volumes-using-pvc-api.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-pvc-api_{context}"]
+
+= Creating data volumes using the PVC API
+
+When you create a data volume using the *pvc* API, the CDI creates the data volume based on what you specify for:
+
+* `accessModes` (`ReadWriteOnce`, `ReadWrtieMany`, `ReadOnlyMany`)
+* `volumeMode` (`Filesystem` or `Block`)
+* `capacity` of `storage` (`5Gi`, for example)
+
+In the following YAML, using the *pvc* API allocates a data volume with a storage capacity of two gigabytes. You specify an access mode of `ReadWriteMany` to enable live migration. Because you know the values your system can support, you specify `Block` storage instead of the default, `Filesystem`.
+
+[source,yaml]
```

Comment:
Same suggestion for a block title, if possible.

Reply:
Done,